app/src/main/java/io/homeassistant/companion/android/settings/shortcuts/views/ManageShortcutsView.kt

```diff
@@ -0,0 +1,456 @@
+                    4 -> viewModel.shortcutLabel4.value
+                    5 -> viewModel.shortcutLabel5.value
+                    6 -> viewModel.shortcutLabelPinned.value
+                    else -> viewModel.shortcutLabel1.value
+                },
+                onValueChange = {
+                    when (index) {
+                        1 -> viewModel.shortcutLabel1.value = it
+                        2 -> viewModel.shortcutLabel2.value = it
+                        3 -> viewModel.shortcutLabel3.value = it
+                        4 -> viewModel.shortcutLabel4.value = it
+                        5 -> viewModel.shortcutLabel5.value = it
+                        6 -> viewModel.shortcutLabelPinned.value = it
+                        else -> viewModel.shortcutLabel1.value = it
+                    }
+                },
+                label = {
+                    Text(
+                        if (index < 6)
+                            "${stringResource(id = R.string.shortcut)} $index ${stringResource(id = R.string.label)}"
+                        else
+                            stringResource(id = R.string.shortcut_pinned_label)
+                    )
+                },
+                modifier = Modifier.padding(top = 16.dp)
+            )
+
+            TextField(
+                value = when (index) {
+                    1 -> viewModel.shortcutDesc1.value
+                    2 -> viewModel.shortcutDesc2.value
+                    3 -> viewModel.shortcutDesc3.value
+                    4 -> viewModel.shortcutDesc4.value
+                    5 -> viewModel.shortcutDesc5.value
+                    6 -> viewModel.shortcutDescPinned.value
+                    else -> viewModel.shortcutDesc1.value
+                },
+                onValueChange = {
+                    when (index) {
+                        1 -> viewModel.shortcutDesc1.value = it
+                        2 -> viewModel.shortcutDesc2.value = it
+                        3 -> viewModel.shortcutDesc3.value = it
+                        4 -> viewModel.shortcutDesc4.value = it
+                        5 -> viewModel.shortcutDesc5.value = it
+                        6 -> viewModel.shortcutDescPinned.value = it
+                        else -> viewModel.shortcutDesc1.value = it
+                    }
+                },
+                label = {
+                    Text(
+                        if (index < 6)
+                            "${stringResource(id = R.string.shortcut)} $index ${stringResource(id = R.string.description)}"
+                        else
+                            stringResource(id = R.string.shortcut_pinned_desc)
+                    )
+                },
+                modifier = Modifier.padding(top = 16.dp, bottom = 16.dp)
+            )
+
+            Text(stringResource(id = R.string.shortcut_type))
+
+            Row {
+                ShortcutRadioButtonRow(viewModel = viewModel, type = "lovelace", index = index)

```

Comment:
These buttons don't work for the pinned shortcuts (because the index is 6)

Reply:
🤦‍♂️  thank you 🙏